src/ERC20Helper.sol

```diff
@@ -1,36 +1,62 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.7;
 
-import { IERC20Like } from "./interfaces/IERC20Like.sol";
+interface IERC20Like {
+
+    function approve(address spender_, uint256 amount_) external returns (bool success_);
+
+    function transfer(address recipient_, uint256 amount_) external returns (bool success_);
+
+    function transferFrom(address owner_, address recipient_, uint256 amount_)
+        external returns (bool success_);
+
+}
 
 /**
  * @title Small Library to standardize erc20 token interactions.
  */
 library ERC20Helper {
 
-    /**************************************************************************************************************************************/
-    /*** Internal Functions                                                                                                             ***/
-    /**************************************************************************************************************************************/
+    /**********************************************************************************************/
+    /*** Internal Functions                                                                     ***/
+    /**********************************************************************************************/
 
-    function transfer(address token_, address to_, uint256 amount_) internal returns (bool success_) {
-        return _call(token_, abi.encodeWithSelector(IERC20Like.transfer.selector, to_, amount_));
+    function safeTransfer(address token_, address to_, uint256 amount_)
+        internal returns (bool success_)
+    {
+        success_
+            = _call(token_, abi.encodeWithSelector(IERC20Like.transfer.selector, to_, amount_));
     }
 
-    function transferFrom(address token_, address from_, address to_, uint256 amount_) internal returns (bool success_) {
-        return _call(token_, abi.encodeWithSelector(IERC20Like.transferFrom.selector, from_, to_, amount_));
+    function safeTransferFrom(address token_, address from_, address to_, uint256 amount_)
+        internal returns (bool success_)
+    {
+        success_ = _call(
+            token_,
+            abi.encodeWithSelector(IERC20Like.transferFrom.selector, from_, to_, amount_)
+        );
     }
 
-    function approve(address token_, address spender_, uint256 amount_) internal returns (bool success_) {
+    function safeApprove(address token_, address spender_, uint256 amount_)
```

Comment:
OZ's version seems more gas efficient: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/utils/SafeERC20.sol#L76 as in most cases it will not need to zero out the approve first. I would switch to that version.

Reply:
Updated